model/pod/podutil_test.go

```diff
@@ -0,0 +1,111 @@
+package pod
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const workingDir = "/data/mci"
```

Comment:
Does this need to be a global constant across all tests in this package? It seems like you could test with any random meaningless directory and set working dir to something like "/some/working/dir" or whatever within each test.